ext/pdo/php_pdo.h

```diff
@@ -49,17 +49,18 @@ extern zend_module_entry pdo_module_entry;
 
 PHP_MINIT_FUNCTION(pdo);
 PHP_MSHUTDOWN_FUNCTION(pdo);
+PHP_RINIT_FUNCTION(pdo);
+PHP_RSHUTDOWN_FUNCTION(pdo);
 PHP_MINFO_FUNCTION(pdo);
 
 ZEND_BEGIN_MODULE_GLOBALS(pdo)
 	zend_long  global_value;
+	HashTable  *pdo_pdbh_to_delete;
```

Comment:
Changing globals would likely breach non core PDO drivers. 7.1 is already in feature freeze, not talking about 7.0, so were a no go. However, from what I see,  pdo_pdbh_to_delete is not required to be accessed by drivers. Thus, using a global ZEND_TLS variable instead of a global would be suitable and wouldn't breach binary compat.

Thanks.